bedrock/firefox/urls.py

```diff
@@ -229,6 +229,8 @@
     page("firefox/privacy/book/", "firefox/privacy/book.html", ftl_files="firefox/privacy/book"),
     # Issue 9957
     page("firefox/more/misinformation/", "firefox/more/misinformation.html", ftl_files="firefox/more/misinformation"),
+    # Firefox for Families campaign, Issue #12004
+    page("firefox/families/", "firefox/campaign/families/index.html", active_locales=["en-US"]),
```

Comment:
Is there a way to place an entire page behind a switch? Should I use `views.py` for that logic?

Reply:
This old issue may be helpful, but it’s also quite old so could be outdated https://github.com/mozilla/bedrock/issues/5868

my understanding is it’s fine to use switches in templates and views, but not in URL files as that would require a server restart.

One option could be to do a redirect in the view to another existing URL if the switch is off. But then again, maybe it’s ok for the page to be live if it’s not linked to from anywhere?

Reply:
Thanks for the context @alexgibson 

Feels like a redirect would be low-effort and good insurance, but I don't have a final url for this yet so it's definitely not going to be linked from anywhere...

@pmac this is an good process question. Should we use a redirect or are we safe to publish a page that won't be linked to until a specific launch date?

Reply:
Resolved to merge into feature branch and open main branch PR when everything is ready

Reply:
IMO it's fine to publish a new page and just not link to it until launch, but we have to be okay with it possibly showing up in search since it will automatically be added to the sitemap. So this is more a question for the stake holders depending on their comfort level with a page being soft-launched like this. 

If we do want to launch it behind a switch, then yes, we could create a custom view for it and raise an [`Http404`](https://docs.djangoproject.com/en/3.2/topics/http/views/#the-http404-exception) error in the view if the switch is off, no real need to redirect a URL that doesn't yet exist IMO.